ledger/src/blockstore_db.rs

```diff
@@ -338,6 +340,7 @@ impl Rocks {
                 access_type: access_type.clone(),
                 oldest_slot,
                 column_options,
+                write_perf_status: BlockstoreRocksDbPerfSamplingStatus::default(),
```

Comment:
nit: rename to "write_batch_perf_status" to help clarify that this is for the `WriteBatch` codepath